scripts/cangen/cangen/config.py

```diff
@@ -0,0 +1,47 @@
+import yaml
+
+
+class Bus:
+    def __init__(self, bus: dict):
+        self.bus = bus
+
+        self.dbc_file_path: str = self.bus.pop("dbcFile")
+        self.bus_name: str = self.bus.pop("busName").capitalize()
+
+        if not self.has_bus_been_consumed():
```

Comment:
don't need an extra method here, just call `if bus` directly, or `if bus.keys()`. Same for the Config class